### PR TITLE
fix(plot): bug in frequency plot start time is out of alignment

### PR DESCRIPTION
**Pull request by hsgwa**
This PR fixes a bug of frequency_plot that the start time of the frequency_plot was misaligned with the correct start time.

JIRA ticket: https://tier4.atlassian.net/browse/T4PB-23472

### CODE REVIEW

**Comment by hsgwa** [comment]
Following errors are not related this PR.
#199 ignores these erros
```
./src/caret_analyze/architecture/architecture.py:156:5: F811 redefinition of unused 'publishers' from line 146
./src/caret_analyze/architecture/architecture.py:161:5: F811 redefinition of unused 'subscriptions' from line 151
```

**Comment by hsgwa** [comment]
@atsushi421 
If you don't mind, could you please review the implementation?

**Review by takeshi-iwanari — DISMISSED** [review]
LGTM!
I confirmed this change fixes the problem, and it works for other cases with Autoware.

@atsushi421 @keita1523 
Could you also review this PR from the view point of code, and merge it if it's okay for you.

File: src/caret_analyze/plot/bokeh/pub_sub_info.py [hunk 1]
```diff
@@ -117,30 +117,46 @@ def _create_frequency_df(
...
-        frequency_df = pd.DataFrame(data={
-            self._get_ts_column_name(pub_sub): ts_series,
-            'frequency [Hz]': freq_series
-        })
         frequency_df = add_top_level_column(frequency_df,
                                             pub_sub.topic_name)
 
         return frequency_df
 
     def _get_earliest_timestamp(
```
Comment (atsushi421) [inline src/caret_analyze/plot/bokeh/pub_sub_info.py:149]: This looks better for `_get_timestamp_range`.

**Review by atsushi421 — COMMENTED** [review]
@hsgwa 
Sorry for the late review.
Please make one small correction.

The parts you have commented on will be fixed by me during the refactoring process.

File: src/caret_analyze/plot/bokeh/pub_sub_info.py (hunk 1 above)
Comment (hsgwa) [reply src/caret_analyze/plot/bokeh/pub_sub_info.py]: addressed in :[43626d3](https://github.com/tier4/CARET_analyze/pull/201/commits/43626d312470c5b83b96289ca9b0372df6fa1af0)

**Review by atsushi421 — APPROVED** [review]
LGTM!

**Pull request merged**